use-cases/invoicing.mdx

```diff
@@ -1,184 +1,189 @@
-
-## 🚀 Quickstart: Create Your First Invoice
+Bring instant, verifiable payments to your invoicing flow. Generate invoices in fiat or crypto, let customers pay in their preferred token across 9 EVM chains and 150+ currencies, and receive automatic reconciliation with on-chain payment detection - no manual tracking, no infrastructure overhead.
 
-<Info>
-**Get your first invoice paid in under 1 minute**
+**What you get:**
+- **Permanent records** - Invoices stored on <Tooltip tip="InterPlanetary File System - a distributed storage network for permanent data">IPFS</Tooltip> create a tamper-proof audit trail
+- **Multi-currency flexibility** - Invoice in USD, get paid in crypto with automatic conversion
+- **Instant reconciliation** - Every invoice automatically linked to its <Tooltip tip="Blockchain-recorded transaction verified by network validators">on-chain</Tooltip> payment
+- **Cross-chain support** - Accept payments across 9 networks in 150+ currencies
+- **Webhook automation** - Real-time payment confirmations for backend integration
 
-*Interactive quickstart experience coming soon*
-</Info>
+## Quickstart
 
 <Steps>
   <Step title="Create Invoice">
-    Generate a payment request with amount, recipient, and payment terms
+    Call the [Create Request endpoint](/api-features/create-requests) with invoice details
   </Step>
-  <Step title="Share Payment Link">
-    Send the secure payment link to your customer
-  </Step>
-  <Step title="Get Paid">
-    Customer pays with their preferred wallet and currency
+  <Step title="Process Payment">
+    Use the payment data to show payment options, then guide the payer through wallet signature and submission
   </Step>
   <Step title="Track Payment">
-    Real-time payment detection and automatic reconciliation
+    Use [webhooks](/api-features/webhooks-events) for real-time payment notifications
   </Step>
 </Steps>
 
-## EasyInvoice Demo
+## Invoicing Demo
+
+Try the complete invoicing workflow with EasyInvoice:
+
+<Card 
+  title="Try EasyInvoice Demo" 
+  href="https://easyinvoice.request.network" 
+  icon="external-link"
+>
+  Create invoices, accept crypto payments, and track payment status - all powered by Request Network API
+</Card>
+
+**What you'll see:**
+- Invoice creation with Request Network
+- Multiple payment currencies and chains
+- Automatic payment detection
+- Real-time status updates via webhooks
 
-See the complete invoicing workflow in action with our demo application:
+<Check>
+**Ready to integrate?** If EasyInvoice meets your needs, you can start using it immediately. For custom requirements, explore the integration options below.
+</Check>
+
+## Integration Approaches
+
+<Info>
+Not sure which approach fits your needs? [See our detailed comparison of integration approaches →](/resources/integration-approaches)
```

Comment:
This link is intentionally left as a placeholder for a future page. Issue #27 has been created to build the Integration Approaches comparison page. The Info callout above this link explicitly notes that the page doesn't exist yet.